v1.10.0

josteinaj opened the pull request:
<!---
@huboard:{"order":53.0,"milestone_order":53,"custom_state":""}
-->

Commit:
Added generated content for table and notes

Commit:
Changes in generated content XSL

Commit:
change from workaround to normal footnote info

Commit:
fixed useful options that don't work in GUI and improved  generated content

josteinaj committed:
bump version to v1.10.0-SNAPSHOT

bertfrees commented:
Can you add some release notes? I usually do that but since the module will be transferred to NLB soon you might as well start doing that. I'm also going to ask to update to the latest stable version of mod-braille once I've released it, and to make sure all the tests pass so that there's nothing left for me to do except uploading the artifacts.

josteinaj commented:
Ok, sure.

Commit:
Rebase with master branch

josteinaj committed:
removed warning

josteinaj commented:
I don't mind seeing the warnings while debugging or developing but I'd rather not have them visible for the end-users. I doubt there will be much of a difference in performance or even maintainability.

I suppose the latest saxon are planned for DP2 v1.10 maybe? We can change back to cleaner code after that's released if we want to.

bertfrees commented:
I know, of course, they shouldn't be visible, but what I mean is any other solution, even filtering the messages in the web UI :), is a nicer solution than this I think.

OK luckily it's just one line that you had to change. If it was a few dozens of these lines I would protest harder, but now I guess it's acceptable.

I'm not so much worried about the performance either.

Anyway, this is really a Saxon issue. They tried to be smart and warn us about things that aren't mistakes. They probably realized that added the configuration property because of that. We could try updating Saxon *now*? By the way I've also been wanting to ask you to try the latest Calabash and see if that has a positive effect on the speed. That could be done at the same time?

josteinaj committed:
updated NEWS. fixed typo: translater->translator

josteinaj commented:
I'm not quite sure how to update calabash/saxon; is it just about updating the version in one of the framework poms?

I updated the NEWS just now. Ammar had already written some but I also added news for v1.9 and v1.8.1 as well.

josteinaj commented:
I've added the ugly hack to mod-braille as well btw ;)

https://github.com/daisy/pipeline-mod-braille/blob/more-messages/pipeline-braille-utils/liblouis-utils/liblouis-utils/src/main/resources/xml/transform/liblouis-block-translate.xsl#L14

https://github.com/daisy/pipeline-mod-braille/blob/more-messages/pipeline-braille-utils/css-utils/css-utils/src/main/resources/xml/transform/block-translator-template.xsl#L14

bertfrees commented:
Thanks.
